code/game/turfs/simulated/floor/asteroid_floors.dm

```diff
@@ -58,6 +58,15 @@
 		if(1)
 			getDug()
 
+/turf/simulated/floor/plating/asteroid/proc/attempt_ore_pickup(obj/item/storage/bag/ore/S, mob/user)
```

Comment:
Is there a reason to scope this only to asteroid turfs? I feel like this is a weird limitation.

Reply:
Just to keep the existing behavior untouched. Ore bags don't auto-pickup on any other kind of turfs at the moment. Might be worth changing in a non-refactor PR.